src/renderer/html_handlebars/hbs_renderer.rs

```diff
@@ -457,7 +458,7 @@ fn id_from_content(content: &str) -> String {
     let mut content = content.to_string();
 
     // Skip any tags or html-encoded stuff
-    let repl_sub = vec![
+    static REPL_SUB: &[&str] = &[
```

Comment:
I'm assuming you meant `const REPL_SUB ...` here. Using `static` means it actually gets embedded in the binary and is typically only done for global mutable objects or if your hardware requires it (like the exception vector when writing a kernel).

I'd also pull it up to the top of the file and name it to something like `HTML_ENCODE_BLACKLIST` (feel free to give it a better name) so it's easy to update later on.

Reply:
Ooop! Right.